src/main/antora/modules/intro/pages/overview/overview-jakartaee-components.adoc

```diff
@@ -0,0 +1,21 @@
+=== Jakarta EE Components
+
+Jakarta EE applications are made up of components. A *Jakarta EE component* is a self-contained functional unit that
+makes use of one or more container services.
+That usage could be as simple as injecting a single dependency
+or responding to REST requests, or as complicated as
+injecting multiple dependencies, querying a database, firing and event, and participating in distributed transactions.
+
+Jakarta EE supports the following components:
+
+* *Web components{empty}footnote:[Not to be confused with the web browser standard, https://developer.mozilla.org/en-US/docs/Web/API/Web_Components[Web Components], or user interface widgets in general, which are often called UI components.]* interact with web standards (HTTPS, HTML, WebSocket, and so on) using Jakarta Servlet, Jakarta Faces, Jakarta WebSocket, and related technologies.
+* Business components implement logic necessary to support the business domain using either *Enterprise bean components (enterprise beans)*{empty}footnote:[Enterprise Beans previously supported persistence via Entity Beans, but that has been deprecated in favor of Jakarta Persistence] or CDI managed bean components.
```

Comment:
Isn't "CDI managed bean components" a little too verbose?
Wouldn't either "CDI beans", "CDI components", "CDI managed beans", or "CDI managed components" be descriptive enough? 
We should avoid any confusion with the deprecated Managed Beans.

Reply:
The updated servlet section uses "CDI managed beans".

Reply:
Then I think "CDI managed beans" is what should be used here as well.